content/guides/forecasts.md

```diff
@@ -93,3 +93,7 @@ For Linear: `forecast(metric_name, ‘linear’, 1, interval='60m', history='1w'
 For Seasonal: `forecast(metric_name, 'seasonal', 1, interval='60m', seasonality='weekly')`, where the options for `seasonality` are: `hourly`, `daily`, and `weekly`.
 
 The start and end times to specify when using the API are the start and end times of the forecast itself. If you want the forecast for the next day you would specify the start to be `now` and the end to be `1 day ahead`.
+
+### Limitations
```

Comment:
I wouldn't call this a "limitation", as those functions just don't make any sense to use with forecasts.

Reply:
Any suggestion for a better term? I think it's still a limitation (even if it's merely preventing something that doesn't make sense to begin with), but I'm happy to change the wording. Not sure what would sound better, though. "Invalid function combinations"?